src/events-header.tsx

```diff
@@ -50,6 +51,10 @@ export default function EventsHeader({ user,
 }: EventsHeaderProps) {
     const navigate = useNavigate();
 
+    const handleTripleClick = useLongPress( () =>  {
```

Comment:
don't want to be picky but then the name is wrong. it should be `const handleLongPress ...